src/pages/Landing.tsx

```diff
@@ -1,36 +1,87 @@
-import { Container, Typography, styled } from "@mui/material";
-import landing from '../assets/landing.png';
 import { useTranslation } from "react-i18next";
+import { Box, Typography, styled, Button, Stack } from "@mui/material";
+import landing from "../assets/landing.png";
 
-const StyledImage = styled('img')`
-    width: 70%;
-    transform: translate(10px, 10px);
-` 
-const StyledTextContainer = styled('div')`
-    display: flex;
-    justify-content: center;
-    align-items: center;
-`
+const StyledImage = styled("img")`
+  display: flex;
+  width: 70vw;
+  height: 109vh;
+  object-fit: fill;
+  position: absolute;
+  right: -20%;
+
+  background-color: #6ca7a3;
+`;
+const StyledTextContainer = styled("div")`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50vw;
+  height: 60vh;
+  object-fit: fill;
+  background-color: #6ca7a3;
+`;
+const blanco = "#efe5e5";
+const azul = "##5ebab3";
+const ColorButton = styled(Button)<ButtonProps>(({ theme }) => ({
+  color: theme.palette.getContrastText(blanco),
+  backgroundColor: blanco,
+  "&:hover": {
+    backgroundColor: azul,
+  },
+}));
 
 export default function Landing() {
-    const { t } = useTranslation();
+  const { t } = useTranslation();
   return (
-  <Container>
-    <StyledTextContainer>
+    <Box component="section">
+      <StyledTextContainer sx={{ position: "absolute", top: "-5%" }}>
         <Typography
-            variant="h1"
-            gutterBottom
-            sx={{
-                color: '#0E0E0E',
-                }}>
-            {t('landing-page.title')}
+          variant="h1"
+          sx={{
+            color: "#f1efef",
+            fontWeight: "700",
+          }}
+        >
+          {t("landing-page.title")}
         </Typography>
-        <StyledImage src={landing} alt="landing" />
-    </StyledTextContainer>
+      </StyledTextContainer>
+
+      {/* Landingpage's Paragraph */}
```

Comment:
Remove this line

Reply:
Can you explain why? I want to make it sure that we both are at the same page because it is a little bit tricky.